tasks/winbuildscripts/Generate-Chocolatey-Package.ps1

```diff
@@ -2,7 +2,11 @@ Param(
     [Parameter(Mandatory=$true,Position=0)]
     [ValidateSet("offline", "online")]
     [String]
-    $installMethod
+    $installMethod,
+
+    [Parameter(Mandatory=$true,Position=1)]
+    [String] 
+    $msiDirectory
```

Comment:
Question: Does setting this as mandatory mean that the "offline" invocation also requires this parameter (even though it's not necessary in that case and it may be breaking the offline variant of the job).

I'm not able to make a better suggestion since I'm not sure I have enough context for what "offline" and "online" actually mean here and I'm a bit clueless about powershell.

Reply:
I forgot about the offline job and will make `Mandatory=$false`. I will create a ticket to delete the offline job and clean up anything that is unused after the chocolatey automation is successful.